lib/iris/tests/stock.py

```diff
@@ -35,6 +35,53 @@
 from iris.coord_systems import GeogCS, RotatedGeogCS
 
 
+def geodetic(shape, with_bounds=True, xlim=None, ylim=None, circular=False):
```

Comment:
There's already a `lat_lon_cube` function in this module which does a very similar (albeit more limited) job. It'd be nicer if we could just extend `lat_lon_cube`.